grading/README.md

```diff
@@ -72,7 +72,7 @@ To earn full credit, you must be identified as "present" in Minutes for at least
 
 - [x] [Hayden](./minutes.md#hayden)
 - [ ] [Cassandra](./pull_request.md#cassandra)
-- [ ] [Parker](./pull_request.md#parker)
+- [x] [Parker](./pull_request.md#parker)
```

Comment:
Can you please change the link to the minutes file.
```suggestion
- [x] [Parker](./minutes.md#parker)
```
It seems you need to make similar changes in the following subsections like reviews, closed_actions, etc.